hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -1223,6 +1231,14 @@ public void start() throws IOException {
 
     DefaultMetricsSystem.initialize("OzoneManager");
 
+    // Start Ratis services
+    if (omRatisServer != null) {
```

Comment:
When would they be null?

Reply:
If ratis is not enabled. This is for the non-HA code path.